bedrock/firefox/templates/firefox/whatsnew/whatsnew-fx60.html

```diff
@@ -0,0 +1,124 @@
+         <iframe id="fxa" scrolling="no" data-src="{{ settings.FXA_IFRAME_SRC }}?action=email&amp;utm_campaign=fxa-embedded-form&amp;utm_medium=referral&amp;utm_source=whatsnew&amp;utm_content=fx-{{ version }}&amp;entrypoint=whatsnew&amp;service=sync&amp;context=iframe&amp;style=chromeless&amp;haltAfterSignIn=true"></iframe>
+       </div>
+     </div>{#-- /#fxa-container --#}
+
+     {#
+       If the user *is* logged in to Sync but does *not* have any mobile devices set up,
+       display either:
+         1) Send to Device widget (if supported by locale)
+         2) QR code for Firefox for mobile
+
+       All visitors in this state also see App & Play store badges.
+     #}
+     <div class="content-container fx-mobile-container" id="fx-mobile-container">
+       <header class="content-header">
+         <h2>{{ _('Take Firefox with You') }}</h2>
+         <p class="tagline">{{ _('Get your bookmarks, history, passwords and other settings on all your devices.') }}</p>
+
+       {% if not show_send_to_device %}
+         {# additional copy for QR code state #}
+         <p>{{ _('Sync up with Firefox on mobile:') }}</p>
+       {% endif %}
+         <ul class="mobile-download-buttons">
+           <li class="android">
+             {{ google_play_button(alt_href='https://mzl.la/2JwvrgL', anchor_attributes={'data-link-type': 'download', 'data-download-os': 'Android', 'id': 'playStoreLink', 'data-mozillaonline-link': settings.GOOGLE_PLAY_FIREFOX_LINK_MOZILLAONLINE}) }}
+           </li>
+           <li class="ios">
+             <a id="appStoreLink" href="https://mzl.la/2JwvrgL" data-link-type="download" data-download-os="iOS">
+               <img src="{{ l10n_img('firefox/ios/btn-app-store.svg') }}" alt="{{ _('Download on the App Store') }}" width="152" height="45">
+             </a>
+           </li>
+         </ul>
+       </header>
+     {% if show_send_to_device %}
+       <div class="send-to-device-wrapper">
+         {{ send_to_device(include_title=False, message_set='fx-50-whatsnew') }}
+       </div>
+     {% else %}
+       <div class="qr-wrapper">
+         <img src="{{ static('img/firefox/whatsnew_60/qrcode-firefox.png') }}" data-mozillaonline-link="{{ static('img/firefox/whatsnew/qrcode-mozillaonline.png') }}" width="350" height="350" alt="" />
+       </div>
+     {% endif %}
+     </div>{#-- /#fx-mobile-container --#}
+
+     {#
+       If the user is logged in to Sync *and* has mobile devices sync'ed, display
+       QR code for Focus and App/Play store badges.
+     #}
+     <div class="content-container focus-container" id="focus-container">
+       <header class="content-header">
+         <h2>{{ _('Firefox Focus') }}</h2>
+         <p class="tagline">{{ _('Blocks ads with trackers. Automatically. Plus, it can even speed up page loading. Get the private browser that has your back.') }}</p>
+         <p>{{ _('Scan the QR code to get started.') }}</p>
```

Comment:
Could this string be shown along side the other QR code state? The added context it provides would be useful, but I guess it depends which locales have this string translated.

Reply:
Adding the string above "Sync up with Firefox on mobile" is a little awkward, but I suppose better than not having it.

(I don't know why this QR code layout is different from that of Focus, but we don't really have time to re-open those discussions.)